perception/ground_segmentation/docs/scan-ground-filter.md

```diff
@@ -30,7 +30,6 @@ This implementation inherits `pointcloud_preprocessor::Filter` class, please ref
 
 | Name                              | Type   | Default Value | Description                                                                   |
 | --------------------------------- | ------ | ------------- | ----------------------------------------------------------------------------- |
-| `base_frame`                      | string | "base_link"   | base_link frame                                                               |
```

Comment:
@badai-nguyen Could you add "input_frame" and "output_frame" as params? Also, I want you to write the limitation that  "input_frame" must be fixed to base_link.